ktor-http/ktor-http-cio/jvm/src/io/ktor/http/cio/websocket/FrameJvm.kt

```diff
@@ -0,0 +1,74 @@
+@file:kotlin.jvm.JvmMultifileClass
+@file:kotlin.jvm.JvmName("FrameKt")
+package io.ktor.http.cio.websocket
+
+import io.ktor.util.*
+import kotlinx.coroutines.*
+import java.nio.*
+
+/**
+ * Frame content
+ */
+val Frame.buffer: ByteBuffer
+    get() = ByteBuffer.wrap(data)
```

Comment:
This is breaking change: the problem is that it doesn't keep state between invocations